lib/room.rb

```diff
@@ -0,0 +1,53 @@
+
+
+class Room
+  attr_reader :id
+  attr_accessor :unavailable_dates
+
+  def initialize(id)
+    @id = id
+    @unavailable_dates = []
+  end
+
+
+  def self.create(number_rooms)
+    rooms = []
+    number_rooms.times do |room_number|
+      rooms << Room.new(room_number + 1)
+    end
+    return rooms
+  end
+
+  def self.list
```

Comment:
Since you're not using this method anywhere else in your program, you can ditch it.